mmv1/products/chronicle/Watchlist.yaml

```diff
@@ -58,9 +60,10 @@ parameters:
       which will become the final component of the watchlist's resource name.
       This value should be 4-63 characters, and valid characters
       are /a-z-/.
+    required: false
     immutable: true
-    url_param_only: true
-    required: true
```

Comment:
Since this is no longer required, can you add a test or example that tests when the field is not set in config?